src/mac/frame/header.rs

```diff
@@ -8,7 +8,8 @@ use byte::{check_len, BytesExt, TryRead, TryWrite, LE};
 use hash32_derive::Hash32;
 
 use super::frame_control::*;
-pub use super::frame_control::{AddressMode, FrameType, FrameVersion, Security};
+pub use super::frame_control::{AddressMode, FrameType, FrameVersion};
+use super::security::*;
```

Comment:
I'm not a fan of these glob imports (except for limited preludes that mostly contain traits), as I think specifying the imported names explicitly makes the code more readable (especially if you don't have IDE support, like when reading code on GitHub).

Not a showstopper, just expressing my preference.